.env

```diff
@@ -13,4 +13,5 @@
 # limitations under the License.
 
 DEBEZIUM_VERSION=1.1
-COMPOSE_HTTP_TIMEOUT=200
\ No newline at end of file
+COMPOSE_HTTP_TIMEOUT=200
+DATABASE_DUMP_PATH=./utils/dbdump # for big data dump use ./utils/big-dbdump
```

Comment:
@kimaina ,thanks here,
but it looks like its same thing as leaving `# for big data dump use   ./utils/big-dbdump ` directly in the compose file. 
i dont see any advantage of using an env variable